src/App.tsx

```diff
@@ -1,19 +1,22 @@
-import { ThemeProvider, useTheme } from "styled-components"
-import { GlobalStyle } from "./styles/global"
-import { defaultTheme } from "./styles/themes/default"
+import { ThemeProvider } from "styled-components";
+import { AvaliationQuestions } from "./components/AvaliationQuestions";
+import { QuestionsMap } from "./components/QuestionsMap";
+import { HomeAvaliations } from "./pages/HomeAvaliations";
+import { GlobalStyle } from "./styles/global";
+import { defaultTheme } from "./styles/themes/default";
 
 function App() {
```

Comment:
acho q seria melhor primeiro pegar o App ja com o Smash configurado e depois concluir essa PR aqui...

Reply:
Tu fala em modificar lá na branch main para q toda vez q fizer o clone já vir modificado?

Reply:
nao nao... depois q essa PR aqui (https://github.com/gamaacademy-labs/assesment-web/pull/11) for mergeada, eh so dar um git pull origin development q da certo